src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -473,18 +487,29 @@ export const CheckoutPayment = ({
           <Description>
             {textKeys.CHECKOUT_PAYMENT_DETAILS_DESCRIPTION()}
           </Description>
-          <div id="dropin-container" ref={adyenRef}></div>
+          {isPaymentConnected ? (
+            <>
+              <PaymentResult>
+                <ThinTick color={colorsV3.gray900} />
+                {textKeys.CHECKOUT_PAYMENT_ADYEN_SETUP_DONE_MESSAGE()}
+              </PaymentResult>
+              <Button
+                fullWidth={true}
+                onClick={() => handleClickCompletePurchase()}
+                disabled={isFormikError}
+              >
+                {textKeys.CHECKOUT_FOOTER_COMPLETE_PURCHASE()}
+              </Button>
+            </>
+          ) : (
+            <div id="dropin-container" ref={adyenRef}></div>
```

Comment:
Is it possible to connect a new card after you connect the initial one? Perhaps it would be safer to keep this mounted and just add "display: none" since we are using a ref?

Reply:
why would you want to do that ? 🤔  If you connected a card the flow pretty much just ends as you get redirect to the finish screen.

Reply:
hehe just thinking of non-linear flows. maybe you realized you wanted to add a different card then the one you added first -- but that's a question for zak to begin with maybe.

Reply:
it is something that we do not consider in the current flow as well. as once payment is connected you are taken to the final page. -- yeah, definitely one for Zak.